code-samples/calendar-event/node/update-calendar-event/index.js

```diff
@@ -0,0 +1,33 @@
+const dotenv = require("dotenv");
+const Nylas = require("nylas");
+
+dotenv.config();
+
+// configuring the Nylas client
+const NylasConfig = {
+    apiKey: `${process.env.API_KEY}`
+}
+
+// initialising the Nylas client with access token
+const nylas = new Nylas(NylasConfig);
+
+// enter the Event ID which you want to update
+const eventId = '<Event ID to be updated>';
+
+nylas.events.find(eventId, {identifier: `${process.env.GRANT_ID}`})
```

Comment:
This code was untested, because the first error I received:

```Error:  NylasApiError: Bearer token invalid
    at APIClient.request (file:///workspaces/nylas-hacktoberfest-2023/code-samples/calendar-event/node/update-calendar-event/node_modules/nylas/lib/esm/apiClient.js:101:29)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  type: 'token.unauthorized_access',
  requestId: 'e248a5de-7acd-4c25-8842-fec3822d01cb',
  providerError: undefined,
  statusCode: 401
  ```
After fixing the following line to:
```
nylas.events.find({eventId, identifier: `${process.env.GRANT_ID}`})
```
I receive the following error:
```
Error:  NylasApiError: calendar_id is a required query parameter.
    at APIClient.request (file:///workspaces/nylas-hacktoberfest-2023/code-samples/calendar-event/node/update-calendar-event/node_modules/nylas/lib/esm/apiClient.js:101:29)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  type: 'invalid_request_error',
  requestId: '40eb4cbe-7ce8-4ca6-8b2d-df78a97e3264',
  providerError: undefined,
  statusCode: 400
}
```

Reply:
I see that parts of the code still need to be updated to v3 and tested against the dashboard.